lib/fizzy/parser_expr.cpp

```diff
@@ -133,8 +133,8 @@ void push_branch_immediates(const ControlFrame& frame, bytes& immediates)
 
 }  // namespace
 
-parser_result<Code> parse_expr(
-    const uint8_t* pos, const uint8_t* end, FuncIdx func_idx, const Module& module)
+parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, uint32_t local_count,
+    FuncIdx func_idx, const Module& module)
 {
     Code code;
```

Comment:
Could move `code.local_count` next to this or even set it in the constructor?

Reply:
Setting in constructor sounds good. You can also make the field const.

There is also option to have `code` object with `local_count` pre-created in `module`. It may be good to leave a TODO comment about this.

Reply:
Added comment for now.